transactionoutbox-core/src/main/java/com/synaos/transactionoutbox/TransactionOutbox.java

```diff
@@ -65,24 +77,15 @@ static TransactionOutboxBuilder builder() {
   ParameterizedScheduleBuilder with();
 
   /**
-   * Identifies any stale tasks queued using {@link #schedule(Class)} (those which were queued more
-   * than supplied {@link TransactionOutboxBuilder#attemptFrequency(Duration)} ago and have been
-   * tried less than {@link TransactionOutboxBuilder#blockAfterAttempts(int)} )} times) and attempts
-   * to resubmit them.
+   * Identifies any stale tasks queued by checking unprocessed entries sorted by date.
```

Comment:
comment has changed a lot, is this intentional?

Reply:
Yes, upcoming ticket of adding javadoc should extend that